### DISCUSSION TITLE
What is the difference or advantage between this software and Windows PowerToys Run

### DISCUSSION THREAD

**Discussion by XIAOZHUXUEJAVA** [Flow-Launcher/Flow.Launcher#2117] [Q&A]
![image](https://user-images.githubusercontent.com/81864199/236394271-1e2f0956-6dc2-49e3-8f25-eb4d30d9446c.png)

**Comment by Novack** [comment]
One of the things that I have not being able to do with PowerToys Run, is the functionality provided by the "Plugin Runner" Flow plugin, which lets you map aliases to arbitrary commands or executables, including parameters. This is useful in many different situations, and has proven to be essential from the times of Launchy. I have no idea why they would not include something like this in PTR. 

Other than that, PTR was made by people who worked on Wox, that is the project from which Flow forked, so they are all pretty similar. 
But Flow has some more features, makes no reasons for privacy concerns or invasive telemetry, and being open source and constantly evolving is usually better than any Microsoft software release.
